### ISSUE TITLE
Cross-site-scripting vulnerability 

### ISSUE THREAD

**Issue by soichih** [rafaelrinaldi/html-meta-tags#4]
This module doesn't seem to escape dangerous HTML characters. It is vulnerable to CSS.

```json
{"name": "\"/><script>alert(\"hacked\");</script><meta hoge=\""}
```

**Comment by soichih** [comment]
Yes, I can send a PR.

Am I correct to assume that, the only character that I need to escape is the double quote (")?

**Comment by rafaelrinaldi** [comment]
Yes, that's the basic rule we should add. Might give [`sanitize-html`](https://www.npmjs.com/package/sanitize-html) a shot.
